packages/runner-python/logging_setup.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from logging.handlers import MemoryHandler
+from time import gmtime
+import sys
+
+
+class LoggingSetup():
+    def __init__(self, temp_log_path, min_loglevel=logging.DEBUG) -> None:
```

Comment:
You mean the startup logfile or the whole logging setup?
When we change log format to json (I assume that's what "object logs" mean) it will be a matter of updating formatter below. Startup log mechanism will work just as it does now.